crates/js-sys/src/lib.rs

```diff
@@ -4792,6 +4792,16 @@ macro_rules! arrays {
                 )
             }
 
+            pub unsafe fn view_mut_raw(ptr: *mut $ty, length: usize) -> $name {
```

Comment:
@alexcrichton do we need it for all arrays: `Int8Array`, `Int16Array`,... `Uint8Array` or just for `Uint8Array`?

Reply:
That's up to you! I think it's fine to add it to all though